org.mersenne.mprime.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.mersenne.mprime
+
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+
+command: mprime-wrapper
+tags:
+  - proprietary
+
+finish-args:
+  - --share=network
```

Comment:
What does it use network for?

Reply:
It's a distributed computing program, it uses network to fetch "work units".